cockatrice/src/gameselector.cpp

```diff
@@ -212,3 +216,17 @@ void GameSelector::processGameInfo(const ServerInfo_Game &info)
 {
     gameListModel->updateGameList(info);
 }
+
+void GameSelector::updateButtonChoices(const QModelIndex &index)
+{
+    if (!index.isValid())
+        return;
+
+    const ServerInfo_Game &game = gameListModel->getGame(index.data(Qt::UserRole).toInt());
+    bool overrideRestrictions = !tabSupervisor->getAdminLocked();
```

Comment:
`bool adminLocked = tabSupervisor->getAdminLocked()`
Return if the admin is locked.
`if (!adminLocked)`
If admin is not locked, then proceed. 
You dont need a custom `overrideRestrictions` name when the original variable says all it needs to.